docs/user/04-00-istio-custom-resource.md

```diff
@@ -16,37 +16,41 @@ This table lists all the possible parameters of the given resource together with
+| **components.pilot.k8s.strategy**                           | object         | Defines the rolling update strategy. To learn more, read about DeploymentStrategy in the [Istio documentation](https://istio.io/latest/docs/reference/config/istio.operator.v1alpha1/#DeploymentStrategy).                                                                                                                                  |
+| **components.proxy**                                        | object         | Defines component configuration for the Istio proxy sidecar.                                                                                                                                                                                                                                                                                |
+| **components.proxy.k8s.resources**                          | object         | Defines [Kubernetes resources requests and limits configuration](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/). To learn more, read about Resources in the [Istio documnetation](https://istio.io/latest/docs/reference/config/istio.operator.v1alpha1/#Resources).                                       |
+| **config**                                                  | object         | Specifies the configuration for the Istio installation.                                                                                                                                                                                                                                                                                     |
+| **config.authorizers**                                      | \[\]authorizer | Specifies the list of external authorizers configured in the Istio service mesh config.                                                                                                                                                                                                                                                     |
+| **config.numTrustedProxies**                                | integer        | Specifies the number of trusted proxies deployed in front of the Istio gateway proxy.                                                                                                                                                                                                                                                       |
+| **experimental**                                            | object         | Defines additional experimental features that can be enabled in experimental builds.                                                                                                                                                                                                                                                        |
```

Comment:
I feel since it's a part of productive API it needs to be in the docs and properly described. I wouldn't bother with too much of the documentation for a feature that is for playing around besides minimal stuff, like API descriptions. That should be completely enough with a notice that it's only working in experimental build.